src/main.rs

```diff
@@ -0,0 +1,25 @@
+ extern crate clap;
+ use clap::{Arg, App};
+extern crate calamine;
+
+fn main() {
+    
+    let matches = App::new("Program2")
```

Comment:
this is not the proper name, what program2??? u should be clear what u r writing think about other while writing code